Source/Tests/VDSO_Emulation.cpp

```diff
@@ -141,12 +236,27 @@ namespace FEX::VDSO {
     },
   };
 
-  void* LoadVDSOThunks(MapperFn Mapper) {
+  void* LoadVDSOThunks(bool Is64Bit, MapperFn Mapper) {
     void* VDSOBase{};
     FEX_CONFIG_OPT(ThunkGuestLibs, THUNKGUESTLIBS);
+    FEX_CONFIG_OPT(ThunkGuestLibs32, THUNKGUESTLIBS32);
+
+    std::filesystem::path ThunkGuestPath{};
+    if (Is64Bit) {
+      ThunkGuestPath = std::filesystem::path(ThunkGuestLibs()) / "libVDSO-guest.so";
+    }
+    else {
+      ThunkGuestPath = std::filesystem::path(ThunkGuestLibs32()) / "libVDSO-guest.so";
+
+      VDSODefinitions[0].ThunkFunction = &FEX::VDSO::x32::time;
```

Comment:
I was surprised to see 32-bit specific code here that seemingly had no equivalent code in the 64-bit path, but turns out you're just overriding the entries that are set up at global scope. Could either use a comment for clarification here, or initialize the global table with `nullptr`s and opt for consistent initialization in both paths instead.

Reply:
I did both